src/adr/0034-distributed-listeners-configuration.md

```diff
@@ -161,3 +163,16 @@ The fact that James is failing forces an admin to react, however 'downgraded' st
 believe James is fully operational, while it is not the case.
 
 We did not chose these options as they imply more development for limited safety gains.
+
+## Deploying a new custom listener
+
+Given a new custom listener, not yet deployed in Distributed James cluster,
+
+To be deploying it, an admin needs to follow these steps:
+
+ - Add the jar in extension-jars folder for each James server
+ - As `extension-jars` is read at instanciation time, no reboot is required to instanciate the new listener.
```

Comment:
it's more a comment for the previous step than a step by itself

Reply:
Let's add a level of indent then...

 - A
      - b